xbmc/utils/LangCodeExpander.cpp

```diff
@@ -43,9 +43,22 @@ struct ISO3166_1
   const char* alpha3;
 };
 
+static constexpr unsigned int ISO_3166_1_ALPHA_2_LEN =
+    2; // ISO-3166-1 Alpha 2 is 2 char in length.  eg: "FR"
+static constexpr unsigned int ISO_3166_1_ALPHA_3_LEN =
+    3; // ISO-3166-1 Alpha 3 is 3 char in length.  eg: "FRA"
+
+struct ISO3166_1A2
+{
+  const char* alpha2UC{nullptr};
+  const char* alpha3UC{nullptr};
+  const char* englishname{nullptr};
+};
+
```

Comment:
Being only used inside this compilation unit, this should be included in an anonymous namespace.

```c++
namespace
{
static constexpr unsigned int ISO_3166_1_ALPHA_2_LEN =
...
} // unnamed namespace
```